src/main/java/com/gd/clinic/entities/Prescription.java

```diff
@@ -0,0 +1,40 @@
+package com.gd.clinic.entities;
+
+import lombok.*;
+import org.hibernate.annotations.Type;
+
+import javax.persistence.*;
+import java.time.OffsetDateTime;
+import java.util.UUID;
+
+@NoArgsConstructor
+@RequiredArgsConstructor
+@Getter
+@Setter
+@ToString
+@Entity
+@Table(name = "Prescriptions")
+public class Prescription {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Type(type = "uuid-char")
+    private UUID id;
+
+    @NonNull
+    @Type(type = "uuid-char")
+    private UUID patientId;
+
+    @NonNull
+    @Type(type = "uuid-char")
+    private UUID treatmentId;
+
+    @NonNull
+    String timePattern;
```

Comment:
please don't forget about encapsulation

Reply:
Must've been real tired when I wrote that, missing access modifiers were added this commit.